test/app/App/Panel/AddToken/index.test.js

```diff
@@ -124,7 +208,73 @@ describe('setting token address', () => {
   })
 })
 
+describe('displaying errors', () => {
+  it('should show the correct error page when the error is "INVALID CONTRACT ADDRESS"', () => {
```

Comment:
same as above, a little more descriptive name can go a long way here. for example `it('should allow the user to navigate back when displaying an error')`

then the one below could be: `it('should allow the user to proceed if we failed to verify the token')`

Reply:
doesn't have to be these verbatim but just the general idea of how to describe what's actually being tested